lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,42 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header" tabindex="0"><%= gettext("My Bookmarks") %></h2>
+
+<% if (bookmarks.length) { %>
+
+    <div class='bookmarks-results-list'>
+        <% _.each(bookmarks, function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item" href="<%= bookmark.get('usage_id') %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>">
```

Comment:
@muhammad-ammar I see why the type and date are being read twice by the screen reader.  If you look at my mockup, I have an aria-labelledby attribute set here on the anchor.  It points to the <h3> (which would need an ID).  This overrides what the screen reader would read if an aria-label or labelledby is not present, giving us more control over what actually gets presented to the user and how it does, which was the tricky part of this exercise.